src/transformers/modeling_tf_ctrl.py

```diff
@@ -104,11 +104,25 @@ def call(self, inputs, training=False):
         q = self.split_into_heads(q, batch_size)
         k = self.split_into_heads(k, batch_size)
         v = self.split_into_heads(v, batch_size)
+
         if layer_past is not None:
             past_key, past_value = tf.unstack(layer_past, axis=0)
             k = tf.concat((past_key, k), axis=-2)
             v = tf.concat((past_value, v), axis=-2)
-        present = tf.stack((k, v), axis=0)
+
+        # to cope with keras serialization
```

Comment:
This is important here: 
I added those `tf.is_tensor` lines to be able to cope with the keras serialization tests. If you know better/cleaner ways of checking this, let me know @LysandreJik @thomwolf